src/index.html

```diff
@@ -152,43 +148,37 @@
             30% off up to ₹150 on orders above ₹400 | Use code JUMBO
           </div>
         </div>
-      </section>
+      </section> -->
 
-      <section class="content-section">
-        <!--  -->
+      <!-- <section class="content-section">
+        
         <div class="sidebar">
           <ul class="sidebar__list"></ul>
         </div>
-        <!--  -->
+        
         <div class="content">
-          <div class="content__heading">
-            <h1 class="content__heading--big">Recommended</h1>
-            <p class="content__heading--small">43 ITEMS</p>
-          </div>
+          <div class="content__heading"></div>
           <ul class="content__list"></ul>
         </div>
-        <!--  -->
+        
         <div class="cart">
-          <div class="content__heading">
-            <h1 class="content__heading--big">Cart</h1>
-            <p class="content__heading--small">2 ITEMS</p>
+          <div class="cart__heading">
+            <h1 class="cart__heading--big">Cart</h1>
+            <p class="cart__heading--small"></p>
           </div>
           <ul class="cart__list"></ul>
-          <div class="cart__summary">
-            <div class="cart__summary__heading">
-              <p class="cart__summary__heading--primary">Subtotal</p>
-              <p class="cart__summary__heading--secondary">
-                Extra charges may apply
-              </p>
-            </div>
-            <p class="cart__summary__price">&#x20B9; 578</p>
-          </div>
+          <div class="cart__summary"></div>
 
-          <button class="cart__btn">Checkout &#8594;</button>
+          <button class="cart__btn cart__btn--red">
```

Comment:
get rid of the comments to make the PR clean.

Reply:
it was an unused file so it is deleted.